lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1163,7 +1194,7 @@ def install_pe_on(install_hosts, opts)
                 host['pe_ver'] ||= Beaker::Options::PEVersionScraper.load_pe_version(host[:pe_dir] || opts[:pe_dir], opts[:pe_version_file])
               end
             end
-            do_install sorted_hosts, opts
+           do_install sorted_hosts, opts
```

Comment:
Looks like there is a space change here. I'm not sure if it's intended, but it looks like the indentation is off by one now.